### PR TITLE
app-misc/rdfind: New package, import from ::guru

### PR DESCRIPTION
**Pull request by tastytea**
Find duplicate files based on their content

Closes: https://bugs.gentoo.org/637384
Signed-off-by: Ronny (tastytea) Gutbrod <gentoo@tastytea.de>

### CODE REVIEW

File: app-misc/rdfind/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	</maintainer>
+	<longdescription lang="en">
+		Rdfind is a command line tool that is useful for compressing backup
+		directories or just finding duplicate files. It compares files based on
+		their content, NOT on their file names.
+	</longdescription>
+	<upstream>
+		<bugs-to>https://github.com/pauldreik/rdfind/issues</bugs-to>
+		<doc>https://rdfind.pauldreik.se/rdfind.1.html</doc>
+	</upstream>
```
[the remark this answers is not in the document]
Comment (tastytea) [reply app-misc/rdfind/metadata.xml]: Fixed/added everything except ~arm.

File: app-misc/rdfind/rdfind-1.4.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+inherit autotools
+
+DESCRIPTION="Find duplicate files based on their content"
+HOMEPAGE="https://github.com/pauldreik/rdfind"
+SRC_URI="https://github.com/pauldreik/rdfind/archive/releases/${PV}.tar.gz -> ${PN}-${PV}.tar.gz"
+S="${WORKDIR}/${PN}-releases-${PV}"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm"
```
[the remark this answers is not in the document]
Comment (juippis) [reply app-misc/rdfind/rdfind-1.4.1.ebuild:15]: Everything else looks good, but I can't merge this while adding the `~arm`. We've got policies for that. So please remove the KEYWORD and let's request it correctly from bugzilla after this has been merged.
https://devmanual.gentoo.org/keywording/index.html#keywording-new-packages

[the remark this answers is not in the document]
Comment (tastytea) [reply app-misc/rdfind/rdfind-1.4.1.ebuild]: Oh sorry, I remembered the manual differently. 😄 Fixed.

**Review by juippis — APPROVED** [review]
Thanks!

**Pull request closed without merging**